<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1318</width>
    <height>758</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="SidebarWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Filter</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Abteilung suchen :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="Abteilung_Edit_comboBox">
         <property name="editable">
          <bool>true</bool>
         </property>
         <property name="currentText">
          <string>Abteilung</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Gruppierung : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="Gruppierung_comboBox">
         <property name="currentText">
          <string/>
         </property>
         <property name="placeholderText">
          <string>Gruppierung</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Fach :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="Fach_comboBox">
         <property name="placeholderText">
          <string>Fach</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Verantwortlicher : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="Verantwortlicher_comboBox">
         <property name="placeholderText">
          <string>Verantwortlicher</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Lagerort :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Lagerort_lineEdit">
         <property name="placeholderText">
          <string>Lagerort</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="Speichern_pushButton">
         <property name="text">
          <string>Speichern</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Abbruch_pushButton">
         <property name="text">
          <string>Abbrechen</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <widget class="QTabWidget" name="tabWidget_Gegenstandsanzeige">
       <property name="geometry">
        <rect>
         <x>-10</x>
         <y>0</y>
         <width>1121</width>
         <height>711</height>
        </rect>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Tab 1</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Tab 2</string>
        </attribute>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1318</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNeues_Inventar"/>
   <addaction name="actionReset"/>
   <addaction name="actionCSVexport"/>
  </widget>
  <action name="actionNeues_Inventar">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ListAdd"/>
   </property>
   <property name="text">
    <string>Neues Inventar</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionReset">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ViewRestore"/>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionCSVexport">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/new/prefix1/Icons/export-to-csv-icon-15.jpg</normaloff>:/new/prefix1/Icons/export-to-csv-icon-15.jpg</iconset>
   </property>
   <property name="text">
    <string>CSVexport</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Icons.qrc"/>
 </resources>
 <connections/>
</ui>
